raiden/messages.py

```diff
@@ -254,7 +255,13 @@ def sender(self) -> Optional[Address]:
         data_that_was_signed = self._data_to_sign()
         message_signature = self.signature
 
-        address = signing.recover_address(data_that_was_signed, message_signature)
+        try:
+            address = to_canonical_address(eth_recover(
+                data=data_that_was_signed,
+                signature=message_signature,
+            ))
+        except ValueError:
```

Comment:
As I said in raiden-libs, please use another exception instead of value_error inside `eth_recover()`

Reply:
Hmm, I think I missed it. Will do